src/contents.ts

```diff
@@ -502,11 +527,26 @@ export class GitHubDrive implements Contents.IDrive {
       if (result === true) {
         return proxiedApiRequest<T>(apiPath, this._serverSettings);
       } else {
```

Comment:
This leaves the Python interface in place so if concerned about security you can still use that instead of storing your token in the JupyterLab application. The settings are separate though which may cause confusion - if you have the Python package installed the JupyterLab settings will be ignored.